test/diff.test.js

```diff
@@ -27,6 +27,7 @@ test('diff: without repairs', opts, function(assert) {
 
```

Comment:
Since you're using mock-aws-s3 you ought to be able to confirm that the content of the backup file that was written is what its supposed to be, and where its supposed to be.